spec/system/composer_spec.rb

```diff
@@ -16,4 +16,60 @@
 
     page.has_css?("#user-card")
   end
+
+  context "in a topic the autocomplete prioritizes" do
```

Comment:
```suggestion
  context "in a topic, the autocomplete prioritizes" do
```

Do we like using commas in these? I don't know!